### ISSUE TITLE
[hackerone] fix tor window size fingerprinting

### ISSUE THREAD

**Issue by diracdeltas** [brave/brave-browser#9341]
we should implement letterboxing (https://blog.torproject.org/new-release-tor-browser-90) or similar to prevent fingerprinting using window sizes in Tor mode

**Comment by diracdeltas** [comment]
waiting on a spec

**Comment by awwfk** [comment]
Hi, I'm someone who writes patches for Tor Browser. I was linked to your browser because of the "Tor support".

I have read in your FAQ that states "if your life depends on it use Tor Browser". Why are you shipping the Tor feature if you are aware that it's broken in its current state? Having choice between Firefox and Chromium is good but not if one of the choices foolishly leads people to think they are secure as with Brave. And we all know people do not read.

Another problem, which would apply if you gain a relevant market share of Tor users (which I honestly do not hope), is that you would be segregating the Tor client network increasing entropy for every client for no reason.

Please disable this until you have fixed your fingerprinting bugs (we couldn't in over 10 years), or better stop wasting your time and help us. I know Brendans personal story with Mozilla but that shouldn't be a reason to try to harm the Tor network. We have no association with Mozilla whatsoever.

**Comment by pes10k** [comment]
Closing in favor of https://github.com/brave/brave-browser/issues/720
